src/main/java/com/massivecraft/factions/config/file/MainConfig.java

```diff
@@ -771,6 +800,7 @@ public boolean isMessageByChunk() {
         @Comment("Displayed prefixes for different roles within a faction")
         private Prefix prefixes = new Prefix();
         private LandRaidControl landRaidControl = new LandRaidControl();
+        private Commands commands = new Commands();
```

Comment:
Maybe this could be migrated up so it's a main section?

Reply:
Just saw your config changes on #1311, will rebase after merge

Reply:
It's merged! :)